web/src/services/i18n/translations/ja.yml

```diff
@@ -415,3 +415,5 @@ Failed to update widget.: ウィジェットのアップデートに失敗しま
 Failed to remove widget.: ウィジェットの削除に失敗しました。
 Failed to update widget alignment.: ウィジェットのアラインメントのアップデートに失敗しました。
 Failed to update the widget align system.: ウィジェットのアラインシステムのアップデートに失敗しました。
+You have passed the maximum value.: You have passed the maximum value.
+You have passed the minimum value.: You have passed the minimum value.
```

Comment:
Shouldn't this be translated to JP?

Reply:
Let's translate all onetime in the future. Because every time we need translate will take much time. I will make a stask on Notion

Reply:
Per @lavalse comment , all translation will be in one patch